src/Microsoft.Docs.Template/Models/attribute/SchemaContentTypeAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Docs.Build
+{
+    public enum SchemaContentType
+    {
+        None,
+        Markdown,
+        Href,
+        Xref,
+    }
+
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+    public abstract class SchemaContentTypeAttribute : Attribute
```

Comment:
>SchemaContentTypeAttribute [](start = 26, length = 26)

consider putting this file to attributes.cs?

Reply:
Will do

---
In reply to: [210817669](https://github.com/dotnet/docfx/pull/3209#discussion_r210817669) [](ancestors = 210817669)